### PR TITLE
Update fail message for `expect_all()` (closes #33)

### LINKED ISSUE #33: Implement class expectations
Something like this:

```r
expect_class(vars(am, vs), "double", data = mtcars)
#> Error: `am`, `vs` not of class `double`
#> Filter: None 
```

### PR DESCRIPTION
**Pull request by kinto-b**
Include reference to variables which fail the check. This makes `expect_all()` useful for performing class expectations.

### CODE REVIEW

**Comment by gorcha** [comment]
Hey! As discussed in #33, best to move `expect_allany()` towards deprecation and replace the innards of `expect_any()`. Also make sure you add a NEWS bullet :wink: (but no need to to bump the dev version again except for big changes)

**Comment by kinto-b** [comment]
Oh yup, I prob should've left this in draft state as it wasn't quite ready yet. Have just added the final bits :)

File: R/expect-generic.R [hunk 1]
```diff
@@ -56,35 +54,77 @@
...
+    names()
+
+  act$result <- apply(act$result, 1, all)
 
+  expect_custom(
+    all(act$result, na.rm = TRUE),
+    glue("{act$lab} has {sum(!act$result, na.rm = TRUE)} records failing \\
+          {act$func_desc} on variable \\
+         {as_english_list(paste0('`', act$var_fail, '`'), 'and/or')}.
+          Vars: {act$var_desc}
```
Comment (gorcha) [inline R/expect-generic.R:100]: Actually don't like `Vars` here. Maybe `Columns` to match the docs? Or `Variable set`?

**Review by gorcha — CHANGES_REQUESTED** [review]
Doing my usual pedant thing...

One big thing is needed though - `expect_make()` uses `expect_allany()` internally. This will need to be updated since we're deprecating (and it'll be good to have the improved failure message across all of the auto generated expectations)

**Comment by kinto-b** [comment]
I love it! Don't ever change :) 

Will make the relevant updates today

File: R/expect-generic.R (hunk 1 above)
Comment (kinto-b) [reply R/expect-generic.R:100]: Hm, I definitely prefer `Vars` to `Columns`. It's just presenting what's passed through as the `vars` argument, so makes sense to me to label it as such. `Variable set` seems like a reasonable middle ground though

Comment (gorcha) [reply R/expect-generic.R:100]: Yeah actually `Vars` is probably fine. Disregard!

Comment (kinto-b) [reply R/expect-generic.R]: Ah, I didn't see this. Already changed to variable set :p Easy to change back. I'm not fussed

Comment (gorcha) [reply R/expect-generic.R]: Variable set it is 😉

**Review by gorcha — APPROVED** [review]

**Pull request merged**